### PR TITLE
Add Outlook CalDav Synchronizer

### PR DESCRIPTION
**Pull request by Macleykun**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://api.github.com/repos/microsoft/winget-pkgs/pulls/95646)

### CODE REVIEW

File: manifests/a/aluxnimm/outlookcaldavsynchronizer/4.3.0/aluxnimm.outlookcaldavsynchronizer.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS0.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: aluxnimm.outlookcaldavsynchronizer
+PackageVersion: 4.3.0
+PackageLocale: en-US
+Publisher: Generalize-IT Solutions OG
```
Comment (Trenly) [inline manifests/a/aluxnimm/outlookcaldavsynchronizer/4.3.0/aluxnimm.outlookcaldavsynchronizer.locale.en-US.yaml:7]:
```suggestion
Publisher: Gerhard Zehetbauer
```
This should match control panel

File: manifests/a/aluxnimm/outlookcaldavsynchronizer/4.3.0/aluxnimm.outlookcaldavsynchronizer.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+PublisherUrl: https://caldavsynchronizer.org/
+PublisherSupportUrl: https://caldavsynchronizer.org/help/issues/
+PrivacyUrl: https://caldavsynchronizer.org/de/datenschutzerklaerung/
+# Author:
+PackageName: Outlook CalDav Synchronizer
```
Comment (Trenly) [inline manifests/a/aluxnimm/outlookcaldavsynchronizer/4.3.0/aluxnimm.outlookcaldavsynchronizer.locale.en-US.yaml:12]:
```suggestion
PackageName: CalDavSynchronizer
```
This should match control panel

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Manual validation complete.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**